tools/turbinia.service

```diff
@@ -0,0 +1,29 @@
+# This file should be in /etc/systemd/system/turbinia.service
+# sudo systemctl enable turbinia
+# sudo systemctl status turbinia
+# sudo systemctl start turbinia
+# sudo systemctl stop turbinia
+# sudo systemctl daemon-reload
+
+[Unit]
+Description=Turbinia Daemon
+Documentation=https://github.com/google/turbinia
+Before=multi-user.target
+After=network.target
+After=remote-fs.target
+
+[Service]
+Type=simple
+PIDFile=/var/run/turbinia-daemon.pid
+User=turbinia
+Group=turbinia
+Restart=always
+RestartSec=10
+ExecStartPre=/bin/sh -c 'sudo systemctl set-environment GOOGLE_APPLICATION_CREDENTIALS=/home/turbinia/turbinia-service-account-creds.json'
+ExecStartPre=/bin/sh -c 'sudo systemctl set-environment TURBINIA_ROLE=$(/usr/bin/python /home/turbinia/turbinia_config_parser.py ROLE)'
```

Comment:
Same comment about the environment, but it looks like in this case this might be the way to go because it does need an exec?

Reply:
What user will this run as?  Does this mean that turbinia will need sudo privs for this as well?

Reply:
Referring to the User and Group attribute above, the processes will be ran as turbinia. However, in sudo, systemctl will run as root (which is required). Yes, turbinia will need sudo privs, which we have already set up in the /etc/sudoers.

Reply:
Per Table 1 at https://www.freedesktop.org/software/systemd/man/systemd.exec.html, the prefix "+" set in any Exec*= denotes "the process is executed with full privileges". Therefore, sudo is not needed here. This means systemd will run Exec* as turbinia/turbinia unless they are denoted with special executiable prefixes. So, I applied the + prefix for ExecStartPre and ExecStartPost lines.